### PR TITLE
put manifest and sw in one file

### LINKED ISSUE #2212: Consolidate manifest and sw tests into one file
**Is your feature request related to a problem? Please describe.**
These are overly specific testing files and are very short. We should test application features together in one place.

**Describe the solution you'd like**
create a subdirectory in the integration folder called /application/ and create and index.js that has all these tests in it.

At the same time rewrite the tests to follow best practices mentioned https://github.com/bbc/simorgh/issues/2046

**Describe alternatives you've considered**
Keep separate atomic tests, as they technical test different things. Doesn't matter too much either way so long as they're in a sensible dir and refactored to follow best practices.

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
Resolves #2212 

**Overall change:** 

- refactor sw and manifest tests, moving the test functionality to one shorter file 

**Code changes:**

- delete manifest and sw files
- refactor new file to be more concise

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: cypress/support/commands/application.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Cypress.Commands.add('testArticleServiceWorker200s', service => {
```
Comment (dr3) [inline cypress/support/commands/application.js]: These commands are far too specific IMO. Commands should share reusable code around our tests. These can only be used in 1 application.

**Review by dr3 — CHANGES_REQUESTED** [review]

File: cypress/integration/application/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+describeForLocalOnly('Application', () => {
+  Object.keys(config).forEach(service => {
+    if (service === 'default') {
+      return;
+    }
+    // All services test sws
+    it(`should return a 200 status code for ${service}'s service worker`, () => {
+      cy.testResponseCodeAndType(
+        `/${service}/sw.js`,
```
Comment (dr3) [inline cypress/integration/application/index.js]: This loses coverage of `news/articles/sw.js` which were using on Live

**Review by dr3 — CHANGES_REQUESTED** [review]

File: cypress/integration/application/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        200,
+        'application/javascript',
+      );
+    });
+  });
+
+  // Once all manifest are done this should be move into the object forEach above
+  ['igbo', 'news/articles', 'pidgin', 'yoruba'].forEach(service => {
```
Comment (jamesbhobbs) [inline cypress/integration/application/index.js:32]: I think this block will also pass on live

**Review by jamesbhobbs — APPROVED** [review]

File: cypress/integration/application/index.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import config from '../../../src/app/lib/config/services';
+import { describeForLocalOnly } from '../../support/limitEnvRuns';
+
+describeForLocalOnly('Application', () => {
+  Object.keys(config).forEach(service => {
+    if (service === 'default') {
```
Comment (dr3) [inline cypress/integration/application/index.js]: This can be `.filter(service => service !== 'default)` above

**Review by dr3 — APPROVED** [review]

**Comment by jamesbhobbs** [comment]
This doesn't need testing

**Pull request merged**